### PR TITLE
app user login, register, logout and respective urls added globaly

### PR DESCRIPTION
**Pull request by azsidhu**
app user login, signup and logout views and urls added as an app visible to all. @ehmadzubair kindly review

### CODE REVIEW

File: digital_benchmark/accounts/forms.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    def clean(self, *args, **kwargs):
+        username = self.cleaned_data.get('username')
+        password = self.cleaned_data.get('password')
+
+        if username and password:
+            user = authenticate(username=username, password=password)
+            if not user:
+                raise forms.ValidationError('This user does not exist')
+            if not user.check_password(password):
```
Comment (ehmadzubair) [inline digital_benchmark/accounts/forms.py]: Don't remember if we need to explicitly do this

File: digital_benchmark/accounts/forms.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            'password',
+            'confirmPassword'
+        ]
+
+    def clean(self, *args, **kwargs):
+        email = self.cleaned_data.get('email')
+        password = self.cleaned_data.get('password')
+        password2 = self.cleaned_data.get('confirmPassword')
+        if password != password2:
+            raise forms.ValidationError("password must match")
```
Comment (ehmadzubair) [inline digital_benchmark/accounts/forms.py]: Passwords*

File: digital_benchmark/accounts/views.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    authenticate,
+    get_user_model,
+    login,
+    logout
+)
+
+from .forms import UserLoginForm, UserRegisterForm
+
+
+def login_view(request):
```
Comment (ehmadzubair) [inline digital_benchmark/accounts/views.py]: Class based view

File: digital_benchmark/accounts/views.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    next = request.GET.get('next')
+    form = UserLoginForm(request.POST or None)
+    # if form.errors.__len__():
```
Comment (ehmadzubair) [inline digital_benchmark/accounts/views.py]: remove this comment

File: digital_benchmark/instagram_benchmark/models.py [hunk 5]
```diff
@@ -3,7 +3,7 @@
 # Create your models here.
 
 class InstagramUser(models.Model):
```
Comment (ehmadzubair) [inline digital_benchmark/instagram_benchmark/models.py]: InstagramProfile

Comment (ehmadzubair) [inline digital_benchmark/instagram_benchmark/models.py]: Do we need to add a FK reference to User?

Comment (azsidhu) [reply digital_benchmark/instagram_benchmark/models.py]: yes, we do need this. I have added this in coming PR

**Pull request merged**